react/components/RadioGroup/index.js

```diff
@@ -20,9 +20,11 @@ class RadioGroup extends React.Component {
           const id = `${name}-${i}`
           return (
             <label
-              className={`db pv2 ph4 br3 ${classNames({
+              className={`db br3 ${classNames({
                 'b--muted-4': !hideBorder,
                 ba: !hideBorder,
```

Comment:
```suggestion
                'b--muted-4 ba pv2 ph4': !hideBorder,
```
You can put the classes together since the condition is the same.